### ISSUE TITLE
Upgrading ubuntu 16.04 xenial to 18.04 bionic successfully.... kinda 

### ISSUE THREAD

**Issue by bahaaaldin214** [dnschneid/crouton#3826]
I had ubuntu xenial 16.04 and I tried to upgrade it to 18.04 becuase it looked really cool in pictures  
```
sudo enter-chroot
sudo apt install update-manager-core
sudo do-release-upgrade
```
but didnt work so I did
```sudo do-release-upgrade -d ```

and it kinda worked.... what you see in the pictures was nothing like that, all I saw at first was just the background picture, than I rebooted my chromebook and than I saw the bar at the top which had activities time and sound/battery/user info thingy, I tried clicking the super(search button) but nothing happened. And I did it one last time than I got some apps working I saw the the same bar but this time when I click the super key it worked, I than had some apps like the dilo browser and xterm, good thing I memorized some command lines, I had to manually install pretty much everything, ubuntu software (which is now called gnome-software) settings, the good gnome terminal and pretty much everything.... beside the rythembox.
Also I had a usb drive that I cant access for some reason,  but I was able to manually mount it using the terminal in 16.04, havent yet tried to in 18.04... I could stilt play my music though. 
Also the software center in both 16.04 and 18.04 gave this error where I couldnt install anything becuase of autho-something problem and I could solve it but I have to do it every time I launch the store which is annoying. 
Some other things I found annoying where when I first installed it tab click was off and I only was able to fix that after installing the setting. And you know how in the pictures you see the/a shelf on the left side? yeah well that wasnt there at all, you could still see it when you click the super key, but I think it will look better if its always there.

Thats about it I know I wasnt supposed to install 18.04 in the first place but hopefully this will help when its officially out for all crouton users

**Comment by DennisLfromGA** [comment]
@bahaaaldin214,

Thanx for that report. Those dist-upgrades don't always go very smoothly.  😵
Since 18.04 is now an LTS release maybe we'll it'll get some crouton luv soon'ish... 😉

Don't forget to rename your chroot if you want: `sudo edit-chroot xenial -m bionic`

*Hope this helps,*
-**DennisLfromGA**

**Comment by bahaaaldin214** [comment]
I did actually and hopefully cause there are so many things not working right now im actually switching back to 16.04..
I found even more bugs which made me the gnome store have to be ran through the terminal ```sudo gnome-software```
you need to manually mount usb drives (forgot the code) but you can still access them through '''media/removable/USB_NAME"
the gnome-terminal doesnt come pre installed but when you manually download it through the terminal it never loads when you click it.

When you go to settings the first thing you see is the wifi section and it says "networkManager needs to be running" same thing with the bluetooth setting but a different error, I never tried to fix the problem since it never really bothered me.

In tweaks there is an option to disable the tocuh bad while typing and I unchecked it to try it out but it wasnt so great, so I tried to turn it back on and it never did.

I tried openening a tar.bz2 (I think tar.gz would be the same) in the terminal and never worked, I tried in the file system and worked but it auto coppied the content in the same place as the tar file which could be useful but sometimes it can get messy.

```chmod +x something.sh``` or ```chmod +777 file.sh``` never worked for me, it doesnt give an error but if I tried
```./file.sh``` or even ```file.sh``` it gives an error some times its perimision denied and some times somthing different but about the same

And the most annoying bug is that mine doesnt come pre-installed.

Okay that last one wasnt a bug, and inst the only thing that wasnt pre-installed (I think I already said that) and I have 2 questions: 

How could you install elementary os 🤔  since if you logout it would come back to chrome os(unless you boot from a usb which I dont intent to since proformance wouldnt be as fast (and it kinda scares me ))

And my second question is how many people _created_ not helped debug or update it just create it in its first form?

**Comment by DennisLfromGA** [comment]
@bahaaaldin214,

> How could you install elementary os 🤔 since if you logout it would come back to chrome os(unless you boot from a usb which I dont intent to since proformance wouldnt be as fast (and it kinda scares me ))

'Elementary OS - freya' is not supported in crouton. There is/was [a pull request for it](https://github.com/dnschneid/crouton/pull/2973) a while back but it has not been updated, approved & merged yet and it looks a little stale at this point.

I think the best place to seek your answer(s) are in [the github/MyDearFreya repo](https://github.com/jeremy-breidenbach/MyDearFreya).

> And my second question is how many people created not helped debug or update _**it**_ just create it in its first form?

I'm not sure what *'it'* you are referring to, sorry. 
But, if you're talking about Ubuntu 18.04 it's really hard to say, you could google it I guess.
If you're talkng about 'crouton' you can look at the ['Insights'](https://github.com/dnschneid/crouton/pulse) tab at the top of this page for some stats and you can pull up crouton's [commit history from day one](https://github.com/dnschneid/crouton/commits/master?after=0c1f52dba52cfdd33a2d578f63ee813969b31de9+1924) if you like (it's almost as old as Chrome OS).

*Hope this helps,*
-**DennisLfromGA**

**Comment by bahaaaldin214** [comment]
woah as old as chrome os and I just got it? wow! Anyway ill do some research maybe I could install it

**Comment by Killa4Reala84** [comment]
I've experimented with this on an hp falco chromebook, using  chroot/crouton, as well as usb-boot. Ex-fat partition seems to work best, and there is no need to fret about a thing as long as you back up your original chrome-os kernel image before attempting usb-boot. I actually think the usb method functions much better and seems to function even smoother and more efficiently than crouton. Also, I have discovered myself that there are some limitations to a chrooted environment if you want to run certain privileged commands.
